tracer/samples/IISInDocker/Dockerfile

```diff
@@ -15,24 +15,16 @@ RUN msbuild /p:Configuration=Release
 
 ### Stage: runtime
 FROM mcr.microsoft.com/dotnet/framework/aspnet:4.8-windowsservercore-ltsc2019 AS runtime
-SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
-
 WORKDIR /inetpub/wwwroot
 COPY --from=build /app/src/. ./
-COPY startup.ps1 .
+COPY install-dd-trace-dotnet.ps1 startup.ps1 ./
+
+# Install the Datadog SDK for .NET using the msi installer
+RUN .\install-dd-trace-dotnet.ps1
+
+# Remove the default ENTRYPOINT "C:\ServiceMonitor.exe w3svc"
+ENTRYPOINT []
 
-# Install the .NET Tracer MSI
 # Note: Normally an IIS reset is also required but 'C:\ServiceMonitor.exe w3svc'
 # will restart IIS and update the environment variables received by the child w3wp processes
-RUN Write-Host 'Determining latest release' ;\
```

Comment:
Is there a reason this can't be done here and instead has to be moved to a file? 🤔 I don't know of a good reason it needs to be?